src/Form/index.tsx

```diff
@@ -1449,58 +1443,59 @@ function Form({
   } else if (anyFinished) {
     return completeState ?? null;
   } else if (!activeStep) {
-    return null;
+    return stepLoader;
   }
 
   const addChin =
     formSettings.showBrand && stepCSS.minHeight !== '100%' && !popupOptions;
   return (
-    <ReactPortal options={popupOptions}>
-      <LoaderContainer
-        showLoader={Boolean(stepLoader)}
-        backgroundColor={stepCSS.backgroundColor ?? 'white'}
-      >
-        {stepLoader}
-      </LoaderContainer>
-      <BootstrapForm
-        {...formProps}
-        autoComplete={formSettings.autocomplete}
-        className={className}
-        ref={formRef}
-        css={{
-          ...stepCSS,
-          ...style,
-          position: 'relative',
-          marginBottom: addChin ? '80px' : '0',
-          display: 'flex',
-          ...(popupOptions ? { borderRadius: '10px' } : {})
-        }}
-      >
-        {children}
-        <Grid step={activeStep} form={form} viewport={viewport} />
-        {popupOptions && (
-          <CloseIcon
-            fill='white'
-            css={{
-              position: 'absolute',
-              top: '17px',
-              right: '17px',
-              cursor: 'pointer'
-            }}
-            onClick={() => popupOptions.onHide && popupOptions.onHide()}
-          />
-        )}
-        {!productionEnv && (
-          <DevNavBar allSteps={steps} curStep={activeStep} history={history} />
-        )}
-        {formSettings.showBrand && (
-          <Watermark
-            addChin={addChin}
-            brandPosition={formSettings.brandPosition}
-          />
-        )}
-      </BootstrapForm>
-    </ReactPortal>
+    <div style={{ position: 'relative', height: '100%', width: '100%' }}>
```

Comment:
will explore whether this div can be conditionally rendered only if the stepLoader is present

but obv don't want to cause remounts of the entire form - so wasn't sure we could do it. but theoretically with a key we can